tests/test_unit_converter.py

```diff
@@ -54,3 +53,20 @@ def test_to_decimals_zero(self):
 
     def test_vat_rate_to_decimal(self):
         assert UnitConverter.vat_rate_to_decimal(2500) == Decimal('0.25')
+
+    def test_price_field_to_decimals(self):
+        price_field = dict(
+            amount=1000,
```

Comment:
Use a more odd amount here to catch any rounding errors.

Reply:
Done!